drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1443,8 +1443,8 @@ void TextureStorage::texture_replace(RID p_texture, RID p_by_texture) {
 		tex_to->tex_id = 0;
 	}
 
-	Vector<RID> proxies_to_update = tex_to->proxies;
-	Vector<RID> proxies_to_redirect = tex_from->proxies;
+	Vector<RID> proxies_to_update = Vector<RID>(tex_to->proxies);
```

Comment:
This is definitely a typo from a recent PR. These should be LocalVectors

Reply:
Changing this to `LocalVector` doesn't matter a whole lot, because it's copying the values around a ton either way (because `LocalVector` cannot make CoW copies).
I thought about resolving this for good with a few tweaks, but gave up due to the risk posed from the change.

I'll change it to `LocalVector` anyway, just so the intention is a bit clearer.